.travis.yml

```diff
@@ -22,13 +22,18 @@ env:
     - SPARK_HOME=$HOME/.cache/spark-versions/$SPARK_BUILD
     - RUN_ONLY_LIGHT_TESTS=True
   matrix:
+    # Run python horovod tests and style first
+    - PYTHON_VERSION=3.6.2 TEST_SUITE="python-tests python/tests/horovod/"
+    - PYTHON_VERSION=3.6.2 TEST_SUITE=pylint
+    - PYTHON_VERSION=3.5.1 TEST_SUITE="python-tests python/tests/horovod/"
+    - PYTHON_VERSION=3.5.1 TEST_SUITE=pylint
+    - PYTHON_VERSION=2.7.13 TEST_SUITE="python-tests python/tests/horovod/"
```

Comment:
We unfortunately have to, since this will also run on python2 supported clusters. I do agree things would be much easier and nicer and neater if we had only python3